javascript/javascript1/week3/cactus.js

```diff
@@ -0,0 +1,47 @@
+const activities = [];
+
+// Function to add activities with the actual date
+function addActivity(activity, duration) {
+    const date = new Date();
+    const today = date.toISOString().split('T')[0]; // Format date as "YYYY-MM-DD"
+    activities.push({ date: today, activity: activity, duration: duration, done: false });
+    console.log(`${activity} for ${duration} minutes added on ${today}`);
+}
+
+
+// Function to check the number of activities today
+function showStatus() {
+    const date = new Date();
+    const today = date.toISOString().split('T')[0]; // Format date as "YYYY-MM-DD"
+    const todayActivities = activities.filter(activity => activity.date === today);
+
+    if (todayActivities.length === 0) {
+        console.log("No activities logged today.");
+    } else {
+        console.log(`You have logged ${todayActivities.length} activities today.`);
+        console.log("Activities:");
+        todayActivities.forEach(act => 
```

Comment:
Nice and easy-to-follow implementation! ⭐  This line stands out a bit, why did you choose to go with a `.forEach` here? 😄